Classes/SVDSPickup.uc

```diff
@@ -5,7 +5,7 @@ class SVDSPickup extends KFWeaponPickup;
 
 defaultproperties
 {
-     Weight=7.000000
+     Weight=8.000000
```

Comment:
Why? SVDS was intended to be used with other 7kg weapons, such as Hunting Rifle.

Reply:
Given the SVDS's ability to stun jasons, high scope zoom, fast reload speed and quick refire time I thought this balance change was necessary as this weapon is a complete upgrade compared to the hunting rifle, and can compete decently with the M14. If you could afford the SVDS there was little reason to buy the hunting rifle as the only difference is bodyshot damage and scope zoom, I thought this change would let hunting rifle remain a viable choice.
It was also to make the SVDS weight closer to the SVD, since I think it's strange there was a weight difference of 4 between the weapons, which will be 2 is SVDS weight is increased and SVD weight is decreased

Reply:
oh, you're right. I completely forgot about the fact that hunting rifle's damage type doesn't suffer damage reduction against large zeds. I think the SVD should get more ammo than it has currently though, it's only 70 right now vs 60 against SVDS, and the SVD's weight prevents you from carrying pretty much any other sharp rifle so I think it should have more base ammo to compensate, maybe that can be considered in the next patch

Reply:
I agree. It could be raised to 80, maybe even more.